src/document.cpp

```diff
@@ -1,13 +1,9 @@
 #include <cstdlib>
 
-#ifdef INCLUDE_DOCTESTS
-#include <doctest/doctest.h>
+#ifdef INCLUDE_TESTS
+#include <catch2/catch.hpp>
```

Comment:
why is this file called document?

Reply:
early versions of the code just had the `document` class in, but have added other functionality and not renamed the file.  not sure maybe `geometry.cpp`?